src/views/Profile/PersonalInfo/PersonalInfo.tsx

```diff
@@ -200,6 +200,19 @@ export const PersonalInfo: React.FC<IEditAuthorProps> = ({ author }) => {
     [errorMessages],
   );
 
+  useEffect(() => {
+    const handleBeforeUnload = (e: BeforeUnloadEvent) => {
+      e.preventDefault();
+      e.returnValue =
+        'Зміни не збережені. Ви впевнені, що хочете залишити сторінку?';
+    };
+
+    window.addEventListener('beforeunload', handleBeforeUnload);
```

Comment:
it looks like this popup will be shown even if there were no changes made. may we want to have it only after some changes are made?

Reply:
Yes, it will be shown when user will interact with page, it can be click in empty space, click in button, etc. I will try to remake it.

Reply:
nicely done